docs/getting-started-on-the-raspberry-pi.md

```diff
@@ -0,0 +1,104 @@
+# Getting Started on the Raspberry Pi 3
+
+This guide assumes you are using the [Raspbian image]. It should work on other
+distributions as well, but some details may be different.
+
+We will compile the driver on the Raspberry Pi itself, of course you can also
+cross compile from your system. 
+
+## Download Driver
+
+Get the latest release from
+https://github.com/Sensirion/embedded-uart-sps/releases (sps30-uart-x.y.z.zip)
+
+```bash
+# on the Raspberry Pi
+$ wget https://github.com/Sensirion/embedded-uart-sps/releases/download/1.0.0/sps30-uart-1.0.0.zip
```

Comment:
I think we should explain that `$` is the user prompt and not part of the command (there's also output later in the file)..

Reply:
Done.